swan-lake/development-tutorials/observability/supported-observability-tools-and-platforms/datadog.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Observe Ballerina programs with DataDog
+description: See how Ballerina supports observability by exposing itself via metrics and tracing to DataDog.
+keywords: ballerina, observability, metrics, tracing, datadog
+permalink: /learn/supported-observability-tools-and-platforms/datadog/
+active: datadog
+intro: The sections below include information about how to observe Ballerina programs with DataDog.
+---
+
+Users can observe Ballerina programs with [DataDog](https://www.datadoghq.com/), which is a comprehensive observability and monitoring platform for cloud-scale applications. It provides developers, IT operations teams, and business users with tools to monitor, troubleshoot, and optimize performance across their entire technology stack, including applications, servers, databases, and services. Both metrics and tracing in Ballerina can be viewed with DataDog.
+
+This [Example](/learn/overview-of-ballerina-observability/#example-a-simple-ballerina-service-to-observe) will be used in the guide. Follow the steps given below to observe Ballerina tracing and metrics in DataDog.
```

Comment:
```suggestion
The sample [shpo service](/learn/overview-of-ballerina-observability/#example-a-simple-ballerina-service-to-observe) will be used in this guide. Follow the steps given below to observe Ballerina tracing and metrics in DataDog.
```

Reply:
Addressed review suggestions and replaced in all occurances.